zboxcore/fileref/fileref.go

```diff
@@ -76,6 +76,7 @@ type Ref struct {
 	Children         []RefEntity      `json:"-" mapstructure:"-"`
 	CreatedAt        common.Timestamp `json:"created_at" mapstructure:"created_at"`
 	UpdatedAt        common.Timestamp `json:"updated_at" mapstructure:"updated_at"`
+	AllocationRoot   string           `json:"WriteMarker"`
```

Comment:
`json:"allocation_root"`

Reply:
https://github.com/0chain/blobber/blob/3c0de5ff2f6732dfd1a7f3a0bf73e4d0a7ba3476/code/go/0chain.net/blobbercore/allocation/file_changer_upload.go#L91

https://github.com/0chain/blobber/blob/2e2fe75f2b7de1e2c96f4e25564f776cc0321c01/code/go/0chain.net/blobbercore/allocation/file_changer_update.go#L90

https://github.com/0chain/blobber/blob/3c0de5ff2f6732dfd1a7f3a0bf73e4d0a7ba3476/code/go/0chain.net/blobbercore/allocation/copyfilechange.go#L134


on the blobber, this is actually allocationRoot

Reply:
https://github.com/0chain/blobber/blob/aa49a30b53415f01e00b36c6e7f814d265ed35dc/code/go/0chain.net/blobbercore/reference/ref.go#L49
`WriteMarker` in Ref table is actually allocation root

Reply:
cc: @lpoli